### DISCUSSION TITLE
How to visualize transformed data as histogram instead of pseudo-color plot?

### DISCUSSION THREAD

**Discussion by denvercal1234GitHub** [ImmuneDynamics/Spectre#96] [Help] [answered]
Hi there, 

Related to discussion #95, would it be possible to visualize the arcsinh transformed data as histograms instead of pseudo-color plots using the make.clour.plot()? 

I find it easier to decide the cofactor that way, which similar to how we would adjust the axes in FlowJo to get Channel values as input. 

Thank you very much for your help!

**Comment by tomashhurst** [comment, marked as answer]
@denvercal1234GitHub we don't have a function within _Spectre_ specifically do make histograms (we have nothing against them, I've just always preferred other plots). However, because the data is stored as a simple data.table, you can use any generic plotting function. For example, check out: http://www.sthda.com/english/wiki/ggplot2-density-plot-quick-start-guide-r-software-and-data-visualization.
